styles/styles.css

```diff
@@ -152,18 +152,17 @@
 /* this selects the sections that have a background image and other blocks
   and sets them as the container for the query below
   sections with background images only get ignored */
-  div.section.section.with-background-image:not(.no-content) {
+  div.section.with-background-image:not(.no-content) {
     container-name: text-container;
     container-type: inline-size;
   }
   
   /* this updates the height of the content inside the section */
-  @container text-container (max-width: 62rem) {
-    body > main > div.with-background-image > picture img,
-    body > main > div.with-background-image > .section-container {
-      height: 110cqh; /* 110% of the container's height */
+  @container text-container (max-width: 991px) {
```

Comment:
By convention, we are following `min-width`. Could you please adjust for the same? Basically make this the default style. BTW, does it make sense to have this apply to wide screens as well?